pkg/client/client.go

```diff
@@ -808,7 +811,7 @@ func (o *OptimizelyClient) getFeatureDecision(featureKey, variableKey string, us
 
 	decisionContext.Variable = variable
 	options := &decide.Options{}
-	featureDecision, err = o.DecisionService.GetFeatureDecision(decisionContext, userContext, options, decide.NewDecisionReasons(options))
+	featureDecision, _, err = o.DecisionService.GetFeatureDecision(decisionContext, userContext, options)
```

Comment:
why `_`, I would suggest, if reason is returned, add it regarding anything is added in reasons or not.

Reply:
I agree, but that function does not do anything with reasons - it does not return them